deploy/sre-pruning/110-pruning-images.CronJob.yaml

```diff
@@ -0,0 +1,24 @@
+---
+apiVersion: batch/v1beta1
+kind: CronJob
+metadata:
+  name: image-pruner
+  namespace: openshift-sre-pruning
+spec:
+  failedJobsHistoryLimit: 5
+  successfulJobsHistoryLimit: 3
+  concurrencyPolicy: Forbid
+  schedule: "* */1 * * *"
+  jobTemplate:
+    spec:
+      template:
+        spec:
+          serviceAccountName: sre-pruner-sa
+          restartPolicy: Never          
+          containers:
+          - name: image-pruner
+            image: quay.io/openshift/origin-cli:v4.0.0
+            args:
+            - /bin/bash
+            - -c
+            - "oc adm prune images --token=$$(cat /var/run/secrets/kubernetes.io/serviceaccount/token) --server=https://$(KUBERNETES_SERVICE_HOST):$(KUBERNETES_SERVICE_PORT) --keep-younger-than=24h --keep-tag-revisions=5 --insecure-skip-tls-verify=true --confirm"
```

Comment:
Nice trick, making the command `bash`! I hadn't thought of that.

Oh, but should it have two $ signs? `--token=$$(cat...` looks possibly wrong.

Reply:
And did you mean for this to be a sub-shell? `--server=https://$(KUBERNETES_SERVICE_HOST)`

I imagine what you meant is `--server=https://${KUBERNETES_SERVICE_HOST}`, which would insert the value of that variable instead.